src/core/include/openvino/core/graph_util.hpp

```diff
@@ -278,4 +279,10 @@ bool replace_output_update_name(Output<Node> node, const Output<Node>& node_inpu
 
 OPENVINO_API
 bool replace_node_update_name(const std::shared_ptr<Node>& target, const std::shared_ptr<Node>& replacement);
+
+OPENVINO_API
+void serialize(const std::shared_ptr<ov::Model>& m,
+               const std::string& xml_path,
+               const std::string& bin_path,
+               ov::pass::Serialize::Version version = ov::pass::Serialize::Version::UNSPECIFIED);
```

Comment:
What is the practical usage for changing version for resulting IR?

Reply:
We have this option to serialize model which was created from source to different IR version (by default v11).
But if Model was created from IR v10, it will be serialized to IR v10